bls/globals.go

```diff
@@ -152,20 +152,19 @@ func EvaluatePolyInEvaluationForm(yFr *Fr, poly []Fr, x *Fr, rootsOfUnity []Fr,
 	MulModFr(yFr, &y, &tmp)
 }
 
-func PolyLinComb(vectors [][]Fr, scalars []Fr) ([]Fr, error) {
+func PolyLinComb(vectors [][]Fr, scalars []Fr, polyDegree int) ([]Fr, error) {
 	l := len(vectors)
```

Comment:
isn't this duplicate with the `polyDegree`? Is there ever a case where there's a 0 that makes the degree lower than the length of these slices, and it breaks the output?

Reply:
Commented above; polyDegree is meant to replace the `vlen:=len(vectors[0])` parameter